docker-compose.ci.yml

```diff
@@ -23,6 +48,17 @@ services:
     volumes:
       - ./build/screenshots:/tmp/screenshots
 
+  api-dummy:
+    image: elifesciences/api-dummy
```

Comment:
want a tag on this?

Reply:
Using `approved` if that sounds right?

Reply:
If you are depending on particular data in your assertions, only a fixed tag can guarantee the same data; but it's usually backward compatible as more content is added to `api-dummy` rather than existing content be changed.

Reply:
This has now moved to `Dockerfile-api-dummy`